src/app/component/profile/pages/edit/edit.component.html

```diff
@@ -0,0 +1 @@
+<p>edit works!</p>
```

Comment:
Both edit.component.html and edit.component.scss files should not exist, they are not imported by the profile edit component.